src/components/NavBar/NavBar.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import './NavBar.css'
+
+class NavBar extends React.Component {
+  constructor (props) {
+    super(props)
+    this.title = props.title
+    this.state = {value: ''}
+    this.data = [{key: 0, value: 'coconut'}, {key: 1, value: 'banana'}, {key: 2, value: 'lime'}]
+    this.handleChange = this.handleChange.bind(this)
+  }
+
+  handleChange (event) {
+    this.setState({value: event.target.value})
+  }
+
+  render () {
+    return (
+      <nav className='NavBar-nav'>
+        <h1 className='NavBar-title'>{this.title}</h1>
```

Comment:
iirc, Hao said this could also be {this.props.title}

Reply:
fixed